### PR TITLE
Vulume slider background

### PR DESCRIPTION
**Pull request by teunreyniers**
The volume slider background is white at the moment
image
this PR makes it transparent so its dark like on the screenshots
image

### CODE REVIEW

**Comment by regseb** [comment]
Thank you for this pull request.

I don't have this problem on Windows 10 or Debian Buster. But there is a bug in Firefox for Android: _[Background of range input control is not transparent](https://bugzilla.mozilla.org/show_bug.cgi?id=1025466)_. Which version of Firefox and which OS do you use?

**Comment by teunreyniers** [comment]
I original found the problem in the nightly build of Firefox (version 66) on Windows. Then I checked it in the regular version of Firefox (version 64) and I thought is was there too but now it just looked fine. So apparently when there is a nightly window from Firefox open and you start the regular version it just opens a new nightly version instead of the regular which made me to think it existed in both  versions. 

In conclusion the problem only occurs in the nightly version 66 and not in the regular version 64. So I guess it just a bug in the nightly of Firefox.

**Comment by regseb** [comment]
I merged your modification because it fixes future versions without impacting current versions. Release 3.3.0 includes your pull request.
